live-examples/js-examples/string/string-raw.html

```diff
@@ -0,0 +1,11 @@
+<pre>
+<code id="static-js">const name = 'Bob';
+
```

Comment:
This `name` declaration and the blank line should be removed.

Reply:
Removed